targets/TARGET_Ambiq_Micro/TARGET_Apollo3/device/i2c_api.c

```diff
@@ -79,6 +119,16 @@ void i2c_init(i2c_t *obj, PinName sda, PinName scl)
 
     // initialization
     iom_init(&obj->i2c.iom_obj);
+#if DEVICE_I2C_ASYNCH
+    // Enable interrupts for NB send to work
+    am_hal_iom_interrupt_enable(obj->i2c.iom_obj.iom.handle, 0x7FFF);
+    if(iom == 0) NVIC_EnableIRQ(IOMSTR0_IRQn);
```

Comment:
astyle should have formatted this differently (using .astyle config file we have in the root)

See the rest of the files, like this

```
   if (AM_HAL_STATUS_SUCCESS != status) {
        return I2C_ERROR_NO_SLAVE;
    }
```
the new changes should be consisted with what it's in the implementation.

Reply:
I was not aware of the .astylerc file. It should be ok now.